pkg/utils/gardener/extension.go

```diff
@@ -0,0 +1,70 @@
+// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package gardener
```

Comment:
What do you think about creating a new package `operator` -> `github.com/gardener/gardener/pkg/utils/gardener/operator` which we can use for all operator related utilities?